content/ja/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Kustomizeを使用してSecretを管理する
+content_type: task
+weight: 30
+description: kustomization.yamlを使用してSecretを作成する
+---
+
+<!-- overview -->
+
+Kubernetes v1.14以降、`kubectl`は[Kustomizeを使ったオブジェクト管理](/docs/tasks/manage-kubernetes-objects/kustomization/)をサポートしています。
+KustomizeはSecretやConfigMapを作成するためのリソースジェネレーターを提供します。
+Kustomizeジェネレーターは、ディレクトリ内の`kustomization.yaml`ファイルで指定します。
+Secretを生成したら、`kubectl apply`でAPIサーバー上にSecretを作成します。
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## Kustomizationファイルを作成する
+
+`kustomization.yaml`ファイルの中で`secretGenerator`を定義し、他の既存のファイルを参照することで、Secretを生成することができます。
+たとえば、以下のkustomizationファイルは`./username.txt`と`./password.txt`を参照しています。
+
+```yaml
+secretGenerator:
+- name: db-user-pass
+  files:
+  - username.txt
+  - password.txt
+```
+
+また、`kustomization.yaml`ファイルの中でリテラルを指定して`secretGenerator`を定義することもできます。
+たとえば、以下のkustomizationファイルには`username`と`password`の2つのリテラルが含まれています。
+
+```yaml
+secretGenerator:
+- name: db-user-pass
+  literals:
+  - username=admin
+  - password=1f2d1e2e67df
+```
+
```

Comment:
[Latest doc](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kustomize/#create-the-kustomization-file) has a method creating Secret by `envs` at third part of this paragraph.
But there is not that part in this PR.
Could you add that?

Reply:
I translated the part.

Reply:
Thank you to add tha part! I checkd that. :smile: